adc_export.py

```diff
@@ -39,6 +39,7 @@ def main(get_config,
         if field_names == ('',):
             field_names = (data_proj.def_field,) + field_names
 
+        print(field_names)
```

Comment:
Do you still need this print? if not, remove it.

Reply:
Done